app/views/content/welcome/my-journey-into-teaching/_introduction.html.erb

```diff
@@ -1,36 +1,176 @@
+
+    <p>
+      When the assistant head suggested teaching science, I wasn’t sure I was cut
+      out for it, I’d never been that academic myself. It turned out my degree
+      covered a lot of the science curriculum, so I decided to give it a go. I’ve
+      never looked back.
+    </p>
+
+    <h2>I discovered that science is all about making connections</h2>
+
+    <p>
+      I’m always showing my students how science relates to everything they’re
+      learning. From enzymes in cooking to the surprising history of a lot of
+      medical discoveries. I love it when they ask a question that I’ve never
+      even thought of.
+    </p>
+  <% elsif subject_category == "english" %>
+    <h2>It all started with my degree in sports science</h2>
+
+    <p>
+      Going to uni wasn't the norm where I'm from, but my teachers always
+      encouraged me. They made me believe that I could achieve anything. Plus,
+      I've always loved English. Believe me, I'm a proper library nerd.
+    </p>
+
+    <h2>I couldn’t wait to teach English</h2>
+
+    <p>
+      After uni, I completed my PGCE and quickly started teaching at an Academy.
+      I threw myself into the job and focused on developing a real reading
+      culture at the school. I knew that improving the literacy rates of every
+      student would make a huge difference, especially with kids who aren’t from
+      privileged backgrounds.
+    </p>
+
+    <h2>I realised that teaching English gives every student a voice</h2>
+
+    <p>
+      When a student can read and talk about their views, it opens them up to so
+      many opportunities. It’s all about having confidence, and I can actually
+      play a part in making that happen.
+    </p>
+
+  <% elsif subject_category == "languages" %>
+    <h2>I knew I wanted to use my degree after graduating</h2>
+
+    <p>
+      Some of my friends were going into office jobs, but that just wasn’t for
+      me. Speaking another language is such a gift – you can connect with other
+      people and cultures. And after 4 years studying Spanish at uni, I wanted to
+      share that passion with others.
+    </p>
+
+    <h2>Teaching abroad was a great experience</h2>
+
+    <p>
+      I taught English for a year whilst living in northern Spain, which really
+      took me out of my comfort zone. Seeing the students' enthusiasm for
+      languages made me want to continue teaching when I returned home so I
+      decided to train to teach at a school in Coventry, where I still work now.
+    </p>
+
+    <h2>I quickly realised how creative I could be in the classroom</h2>
+
+    <p>
+      There are loads of fun ways to bring languages to life. For a challenging
+      GCSE class, speed dating in Spanish really got their attention! For the
+      younger students, they love pretending to order off the restaurant menus
+      that I bring back from my holidays in Spain.
+    </p>
```

Comment:
Missing content to be added in after 'holidays in Spain.'

<h2>What may surprise you?</h2>
<p>Teaching Spanish is about much more than grammar and vocab, it’s about exploring another culture and sharing different ideas.</p>

<h2>It’s not always easy</h2>
<p>Every student is different and there will be some in each class who aren’t as passionate about learning languages and they don’t understand why it’s important. I always try to use different techniques or games to get them excited.</p>

<h2>This year I’m taking on new responsibilities</h2>
<p>I’m going to be leading on new tech initiatives at my school, to support students to learn in new and exciting ways. I think being able to speak another language has made me a natural communicator, which really helps in these types of roles.</p>